crates/xilem_html/src/element/mod.rs

```diff
@@ -34,7 +38,7 @@ impl<El, ViewSeq> Element<El, ViewSeq> {
             fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
                 write!(f, "<{}", self.0.name)?;
                 for (name, value) in &self.0.attributes {
-                    write!(f, " {name}=\"{value}\"")?;
+                    write!(f, " {name}=\"{}\"", value.serialize())?;
```

Comment:
This piece of code relies on the invariant that `self.0.attributes` never contains `Null`.